fs/artifact_fetcher.go

```diff
@@ -66,32 +71,75 @@ func newArtifactFetcher(refspec reference.Spec, localStore, remoteStore content.
 	}, nil
 }
 
+// All three (jitter,backoffStrategy,retryStrategy) are copied over from
+// https://github.com/awslabs/soci-snapshotter/blob/main/fs/remote/resolver.go.
+// TODO: Consolidate retry/http logic into its own package.
+func jitter(duration time.Duration, divisor int64) time.Duration {
+	return time.Duration(rand.Int63n(int64(duration)/divisor) + int64(duration))
+}
+
+func backoffStrategy(min, max time.Duration, attemptNum int, resp *http.Response) time.Duration {
+	delayTime := rhttp.DefaultBackoff(min, max, attemptNum, resp)
+	return jitter(delayTime, 8)
+}
+
+func retryStrategy(ctx context.Context, resp *http.Response, err error) (bool, error) {
```

Comment:
can we just create a `util/http/retry.go` and move these three there from https://github.com/awslabs/soci-snapshotter/blob/main/fs/remote/resolver.go, instead of making a copy.